Clojure-Functions.md

```diff
@@ -0,0 +1,82 @@
+# Clojure - More on Functions
+Functions! They're pretty important. It's pretty difficult to do anything without
+a function. They're integral to any language, but especially Clojure, since it's
+a functional programming language that rejects object-oriented design. Let's
+learn some more about them!
+
+## Arity
+'Arity' refers to the number of arguments that your function expects.
```

Comment:
Is fine, I'm just used to double quotes, no need to change for me, as far as I know they both work for quotes.

I only use single quotes when coding and double quotes were already used. FOr articles I normally use `inline code block` for code, **key terms**, and _something just italic_ for keywords.